pages/events/index.vue

```diff
@@ -199,7 +204,17 @@ const filteredEvents = computed(() => {
     });
   }
 });
-
+function altTextSorting(event: any) {
```

Comment:
I think this name of function doesn't express what it's doing, I think something like `getEventAltText` is more semantic